tidy3d/components/tcad/data/sim_data.py

```diff
@@ -84,6 +85,12 @@ class DeviceCharacteristics(Tidy3dBaseModel):
         "is given in Ohms. Note that in 2D the resistance is given in :math:`\\Omega \\mu`.",
     )
 
+    ac_current_voltage: Optional[FreqPotentialDataArray] = pd.Field(
+        None,
+        title="AC current vs voltage and frequency",
+        description="Complex small-signal current I(v, f) as a FreqPotentialDataArray.",
```

Comment:
Again, no need to say `as a FreqPotentialDataArray` here. Since the field type is `Optional[FreqPotentialDataArray]`, this information will already appear in the parsed docstring, including a clickable link to the type.